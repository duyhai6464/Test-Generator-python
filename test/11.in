35
422 133
900 333
338 850
340 450
129 182
968 853
986 824
631 549
282 423
507 991
640 483
422 794
548 676
303 543
560 991
570 776
426 298
115 310
154 514
730 657
762 927
580 281
442 421
444 746
183 763
878 253
290 232
376 673
791 332
373 108
857 120
478 326
921 892
969 846
901 410
