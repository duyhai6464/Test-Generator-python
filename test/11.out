Aco
Aco
Aco
Aco
Aco
Kbich
Aco
Kbich
Aco
Aco
Kbich
Kbich
Aco
Aco
Kbich
Kbich
Kbich
Kbich
Aco
Kbich
Aco
Aco
Aco
Kbich
Aco
Aco
Aco
Aco
Aco
Kbich
Aco
Aco
Aco
Aco
Aco
Kbich
Aco
Kbich
Kbich
Aco
Aco
Aco
Aco
Aco
Aco
Kbich
Aco
Aco
Aco
Aco
Aco
Aco
Kbich
Kbich
Aco
Aco
Kbich
Aco
Kbich
Kbich
Kbich
Kbich
Aco
Aco
Kbich
Kbich
Aco
Kbich
Kbich
Aco
Aco
Aco
Kbich
Aco
Kbich
Kbich
Kbich
Aco
Kbich
Kbich
Aco
Kbich
Aco
Aco
Kbich
Kbich
Kbich
Aco
Aco
Aco
Aco
Kbich
Aco
Kbich
Kbich
Kbich
Aco
Aco
Kbich
Aco
